src/main/java/org/folio/rest/util/ResourcePathResolver.java

```diff
@@ -49,6 +51,8 @@ private ResourcePathResolver() {
     apis.put(LEDGER_ROLLOVERS_PROGRESS_STORAGE, "/finance-storage/ledger-rollovers-progress");
     apis.put(GROUPS, "/finance-storage/groups");
     apis.put(TRANSACTIONS, "/finance-storage/transactions");
+    apis.put(BATCH_TRANSACTIONS, "/finance/batch-all-or-nothing");
```

Comment:
do we need api for business module here? all other endoints pointed only to storage module except /finance/expense-classes

Reply:
This is used for testing the API. The others are often using TestEntities to get the finance endpoint, but this is not suitable for the case of batch processing.